src/components/MapLegend.vue

```diff
@@ -398,11 +398,11 @@ $buttonActiveTextColor: #fff;
   padding: 10px;
   align-items: center;
   .siteDot{
-    background: #006633;
-    width:10px;
-    height: 10px;
+    background: #000000;
+    width:6px;
+    height: 6px;
     border-radius: 5px;
-    border: 1px solid #014421;
+    border: 1px solid #000000;
```

Comment:
didn't make this white as the white border on the white background of the legend ends up coming out kind of fuzzy on my screen, figuring this is cleaner/sharper looking